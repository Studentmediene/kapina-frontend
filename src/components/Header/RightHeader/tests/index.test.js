import React from 'react';
import { shallow } from 'enzyme';

import { RightHeader } from '../';

describe('<RightHeader />', () => {
  it('renders correctly', () => {
    const tree = shallow(<RightHeader />);
    expect(tree).toMatchSnapshot();
  });
});
