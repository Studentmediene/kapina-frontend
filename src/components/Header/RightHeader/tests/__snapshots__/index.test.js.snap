// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RightHeader /> renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="headerPlayButtonContainer"
  >
    <Connect(HeaderPlayButton) />
  </div>
  <div
    className="navigationListContainer"
  >
    <NavList
      links={
        Array [
          Object {
            "path": "/sendeplan",
            "title": "Sendeplan",
          },
        ]
      }
    />
  </div>
</div>
`;
